modules/registrars/openprovider/OpenProvider/API/ApiV1.php

```diff
@@ -171,6 +171,22 @@ private function log(string $cmd, array $request, Response $response): void
             ],
         ];
 
+        // Check if Message contains both "Invalid" and "country code" phrases
+        if (
+            strpos($response->getMessage(), "Invalid") !== false &&
+            strpos($response->getMessage(), "country code") !== false
```

Comment:
can't we check whole error message here?

Reply:
`"Invalid country code!"` Received at the time the issue was reproduced. But the message may contain additional information. Ex: "Invalid country code!...other info...". However invalid country code issues should have `"Invalid"` and `"country code"` phrases.

Reply:
mmm okay, then can;t you test both at once like
`strpos($response->getMessage(), "Invalid country code") !== false`

Reply:
This change has been added